WooCommerce/Classes/ViewRelated/Orders/OrdersSplitViewWrapperController.swift

```diff
@@ -61,6 +61,15 @@ private extension OrdersSplitViewWrapperController {
         ordersSplitViewController.viewControllers = [ordersNavigationController, emptyStateViewController]
     }
 
+    func handleCollapsingSplitView(splitViewController: UISplitViewController) -> UISplitViewController.Column {
+        let secondaryColumnNavigationController = splitViewController.viewController(for: .secondary) as? UINavigationController
+        if let navigationController = secondaryColumnNavigationController,
```

Comment:
nit: maybe this can be merged to one line:
```suggestion
        if let navigationController = splitViewController.viewController(for: .secondary) as? UINavigationController,
```

Reply:
👍 Updated in e682917e315016f568b7a02136727039f8540bb3.